hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/mover/TestMover.java

```diff
@@ -872,7 +872,6 @@ void initConfWithStripe(Configuration conf) {
   public void testMoverWithStripedFile() throws Exception {
     final Configuration conf = new HdfsConfiguration();
     initConfWithStripe(conf);
-
```

Comment:
I would avoid removing this empty line (and 987).

Reply:
Thanks for the review @goiri. The reason why I had to remove at least 2 lines is because of checkstyle warning, this is part of build#1 :) 

<img width="1546" alt="Screenshot 2021-09-16 at 11 23 31 AM" src="https://user-images.githubusercontent.com/34790606/133557305-2557ae2c-958f-41b8-8c7f-cd127c7e4685.png">

Reply:
Are you fine with this @goiri? After the recent commit, the method has exactly 150 lines, one more line will create checkstyle warning.
Thanks

Reply:
If the goal is to avoid the 150 lines issue, I would prefer trying to make refactor the method a little instead of just removing lines that hurt readability. Not strongly opposed but not ideal.

Reply:
> Not strongly opposed but not ideal.

I agree with your review, addressed in the latest commit. No change in logic, just the refactor as you have suggested.